gnowsys-ndf/gnowsys_ndf/ndf/static/ndf/scss/_app_styles.scss

```diff
@@ -460,50 +460,96 @@ body{
 
 
```

Comment:
Colors used in footer should be accordingly  particular theme colors.Right now you have used Clix style colors.